corehq/apps/enterprise/enterprise.py

```diff
@@ -383,3 +389,85 @@ def _get_individual_export_rows(self, exports, export_line_counts):
             )
 
         return rows
+
+
+class EnterpriseSMSReport(EnterpriseReport):
+    title = gettext_lazy('SMS Sent')
+    MAX_DATE_RANGE_DAYS = 90
+
+    def __init__(self, account, couch_user, start_date=None, end_date=None, num_days=30):
+        super().__init__(account, couch_user)
+
+        if not end_date:
+            end_date = datetime.utcnow()
+        elif isinstance(end_date, str):
+            end_date = datetime.fromisoformat(end_date)
+
+        if start_date:
+            if isinstance(start_date, str):
+                start_date = datetime.fromisoformat(start_date)
+            self.datespan = DateSpan(start_date, end_date)
+            self.subtitle = _("{} to {}").format(
+                start_date.date(),
+                end_date.date(),
+            )
+        else:
+            self.datespan = DateSpan(end_date - timedelta(days=num_days), end_date)
+            self.subtitle = _("past {} days").format(num_days)
+
+        if self.datespan.enddate - self.datespan.startdate > timedelta(days=self.MAX_DATE_RANGE_DAYS):
+            raise TooMuchRequestedDataError(
+                _('Date ranges with more than {} days are not supported').format(self.MAX_DATE_RANGE_DAYS)
+            )
+
+    def total_for_domain(self, domain_obj):
+        query = SMS.objects.filter(
+            domain=domain_obj.name,
+            direction=OUTGOING,
+            date__gte=self.datespan.startdate,
+            date__lt=self.datespan.enddate_adjusted
+        )
+
+        return query.count()
+
+    def create_count_subquery(self, **kwargs):
+        return Coalesce(
+            Subquery(
+                SMS.objects.filter(
+                    domain=OuterRef('domain'),
+                    date__gte=self.datespan.startdate,
+                    date__lt=self.datespan.enddate_adjusted,
+                    **kwargs
+                )
+                .values('domain')
+                .annotate(count=Count('pk'))
+                .values('count')
+            ),
+            0
+        )
+
+    @property
+    def headers(self):
+        headers = super().headers
+        headers = [_('Project Space Name'), _('# Sent'), _('# Received'), _('# Errors')]
```

Comment:
Simplify the header to be `Project Space`

Reply:
8310efd046ed9a70859397466b507bf5e1b82a21